packages/table/test/regionsTests.ts

```diff
@@ -238,7 +238,7 @@ describe("Regions", () => {
         ];
 
         const joinedGroups = Regions.joinStyledRegionGroups([Regions.row(2)], myGroups, undefined);
-        expect(joinedGroups).to.have.lengthOf(2);
+        expect(joinedGroups).to.have.to.have.lengthOf(2);
```

Comment:
**nit:** duplicate chain for assertions in this file:
```suggestion
        expect(joinedGroups).to.have.lengthOf(2);
```

Reply:
Good catch. Fixed